doc/languages-frameworks/javascript.section.md

```diff
@@ -196,6 +196,7 @@ buildNpmPackage rec {
 * `npmDepsHash`: The output hash of the dependencies for this project. Can be calculated in advance with [`prefetch-npm-deps`](#javascript-buildNpmPackage-prefetch-npm-deps).
 * `makeCacheWritable`: Whether to make the cache writable prior to installing dependencies. Don't set this unless npm tries to write to the cache directory, as it can slow down the build.
 * `npmBuildScript`: The script to run to build the project. Defaults to `"build"`.
+* `dontNpmBuild`: Option to disable running the build script, set to `true` if it's missing in the project. Defaults to `false`.
```

Comment:
I'd argue we should also mention you can disable this by setting `buildPhase` explicitly. cc @lilyinstarlight for her thoughts

Reply:
That's not a bad idea. I'm not sure how best to phrase or whether it should even go here or somewhere else in the docs, though

We are also missing docs for `dontNpmInstall`